packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -50,26 +50,49 @@
         class="resource-table-resource-wrapper"
         :class="[{ 'resource-table-resource-wrapper-limit-max-width': hasRenameAction(item) }]"
       >
-        <oc-resource
-          :key="`${item.path}-${resourceDomSelector(item)}-${item.thumbnail}`"
-          :resource="item"
-          :is-path-displayed="arePathsDisplayed"
-          :parent-folder-name-default="getDefaultParentFolderName(item)"
-          :is-thumbnail-displayed="areThumbnailsDisplayed"
-          :is-extension-displayed="areFileExtensionsShown"
-          :is-resource-clickable="isResourceClickable(item.id)"
-          :folder-link="folderLink(item)"
-          :parent-folder-link="parentFolderLink(item)"
-          @click="emitFileClick(item)"
-        />
-        <oc-button
-          v-if="hasRenameAction(item)"
-          class="resource-table-edit-name"
-          appearance="raw"
-          @click="openRenameDialog(item)"
-        >
-          <oc-icon name="edit-2" fill-type="line" size="small" />
-        </oc-button>
+        <template v-if="item.processing">
```

Comment:
Throtling the network heavily and uploading a bigger file (GPRS + 12MB in my case) meant that the whole ResourceTable (incl. existing resources) turned into a loading state, I'd have imagined some skeleton row to be present for the currently uploading resource?

Reply:
unfortunately thats the case, we increase the polling every time by a factor... this will be significant better once we have sockets in place. But you are right, i implement a configuration option to setup that factor